src/main/kotlin/io/snyk/plugin/net/UserService.kt

```diff
@@ -8,8 +8,12 @@ import retrofit2.http.GET
  * An endpoint to get information about users and groups.
  */
 interface UserService {
-    @GET("user/me")
+    @GET(apiName)
     fun userMe(): Call<User>
+
+    companion object {
+        const val apiName = "user/me"
+    }
```

Comment:
@ArtsiomCh , as we discussed let's remove companion object (Retrofit logs full endpoint) to keep the interface simpler to read.